### PR TITLE
Tests for CASSANDRA-7066 (standalone sstablelister)

### PR DESCRIPTION
**Pull request by stef1927**
We've introduced a new standalone tool (sstablelister) for CASSANDRA-7066 and these are some basic tests for it.

### CODE REVIEW

File: sstablelist_test.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        else:
+            args = [ tool_bin, '--type', type, ks, table]
+
+        p = subprocess.Popen(args, env=env, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+        (stdin, stderr) = p.communicate()
+
+        if len(stderr) > 0:
+            debug("Error invoking sstablelister : %s " %(stderr,))
+            assert false
```
Comment (ptnapoleon) [inline sstablelist_test.py]: This should be `assert False`, not `assert false`.

**Comment by ptnapoleon** [comment]
Other than the incorrect assert, +1

File: sstablelist_test.py (hunk 1 above)
Comment (aboudreault) [inline sstablelist_test.py]: suggestion: would it be clearer to do:
```
debug(stderr)
assert False, "Error invoking sstablelister"
```

Will that make the error more obvious in the exception output? not a strong opinion, it's up to you.

Comment (ptnapoleon) [inline sstablelist_test.py]: I prefer your suggestion, Alan.

**Comment by stef1927** [comment]
Thanks for picking it up, I fixed it with Alan's suggestion and squashed it: commit 4486c1e.

**Pull request merged**